proposals/Level-1.md

```diff
@@ -265,25 +229,30 @@ else
 end
 ```
 
-The conditional query of an exception succeeds when the exception value on the
-top of the stack has the corresponding exception tag (defined by `except_index`).
+In the first form, the instructions between the `if_except` and 'end' define the
+`then block`. In the second form, the instructions between the `if_except` and
+`else` define the `then block`, while the instructions between the `else` and
+the `end` define the `else block`.
```

Comment:
@sunfishcode pointed out that this optional else block complicates decoding. We should consider making this not optional (#54). This is an issue for another PR though.